tests/ui_tests/test_admin_page.py

```diff
@@ -347,4 +352,73 @@ def test_delete_group_from_groups_page(self, app_fs):
         page.create_custom_group(self.custom_group.name, self.custom_group.description, self.custom_group.users)
         page.select_all_groups()
         page.click_delete_button()
-        assert len(page.table.get_all_rows()) == 0, "There should be 0 groups"
+        with allure.step('Check that group has been deleted'):
+            assert len(page.table.get_all_rows()) == 0, "There should be 0 groups"
+
+
+class TestAdminPolicyPage:
+    """Tests for the /admin/policies"""
+
+    custom_policy = AdminPolicyInfo(
+        name="Test policy name",
+        description="Test policy description",
+        role="ADCM User",
+        users="admin,status",
+        groups=None,
+    )
+
+    def test_open_by_tab_admin_policies_page(self, app_fs):
+        """Test open /admin/policies from left menu"""
+
+        intro_page = AdminIntroPage(app_fs.driver, app_fs.adcm.url).open()
+        policies_page = intro_page.open_policies_menu()
+        policies_page.wait_page_is_opened()
```

Comment:
`open_policies_menu` is waiting for the page to be opened, do we need to recheck?

Reply:
removed